src/dom_types.rs

```diff
@@ -273,7 +273,84 @@ make_attrs! {
+    GradientUnits => "gradientUnits", Hanging => "hanging",
+    HorizAdvX => "horiz-adv-x",
+    HorizOriginX => "horiz-origin-x", Ideographic => "ideographic",
+    ImageRendering => "image-rendering", In => "in", In2 => "in2",
+    Intercept => "intercept", K => "k", K1 => "k1",
+    K2 => "k2", K3 => "k3", K4 => "k4",
+    KernelMatrix => "kernelMatrix", KernelUnitLength => "kernelUnitLength", Kerning => "kerning",
+    KeyPoints => "keyPoints", KeySplines => "keySplines", KeyTimes => "keyTimes",
+    LengthAdjust => "lengthAdjust", LetterSpacing => "letter-spacing",
+    LightingColor => "lighting-color", LimitingConeAngle => "limitingConeAngle", Local => "local",
+    MarkerEnd => "marker-end", MarkerMid => "marker-mid", MarkerStart => "marker-start",
+    MarkerHeight => "markerHeight", MarkerUnits => "markerUnits", MarkerWidth => "markerWidth",
+    Mask => "mask", MaskContentUnits => "maskContentUnits", MaskUnits => "maskUnits",
+    Mathematical => "mathematical",
+    Mode => "mode",
+    NumOctaves => "numOctaves", Offset => "offset",
+    Opacity => "opacity", Operator => "operator", Order => "order",
+    Orient => "orient", Orientation => "orientation", Origin => "origin",
+    Overflow => "overflow", OverlinePosition => "overline-position", OverlineThickness => "overline-thickness",
+    Panose1 => "panose-1", PaintOrder => "paint-order",
+    PathLength => "pathLength", PatternContentUnits => "patternContentUnits", PatternTransform => "patternTransform",
+    PatternUnits => "patternUnits", Ping => "ping", PointerEvents => "pointer-events",
+    Points => "points", PointsAtX => "pointsAtX", PointsAtY => "pointsAtY",
+    PointsAtZ => "pointsAtZ", PreserveAlpha => "preserveAlpha", PreserveAspectRatio => "preserveAspectRatio",
+    PrimitiveUnits => "primitiveUnits", R => "r", Radius => "radius",
+    ReferrerPolicy => "referrerPolicy", RefX => "refX", RefY => "refY",
+    RenderingIntent => "rendering-intent", RepeatCount => "repeatCount",
+    RepeatDur => "repeatDur", RequiredExtensions => "requiredExtensions", RequiredFeatures => "requiredFeatures",
+    Restart => "restart", Result => "result", Rotate => "rotate",
+    Rx => "rx", Ry => "ry", Scale => "scale",
+    Seed => "seed", ShapeRendering => "shape-rendering", Slope => "slope",
+    Spacing => "spacing", SpecularConstant => "specularConstant", SpecularExponent => "specularExponent",
+    Speed => "speed", SpreadMethod => "spreadMethod", StartOffset => "startOffset",
+    StdDeviation => "stdDeviation", Stemh => "stemh", Stemv => "stemv",
+    StitchTiles => "stitchTiles", StopColor => "stop-color", StopOpacity => "stop-opacity",
+    StrikethroughPosition => "strikethrough-position", StrikethroughThickness => "strikethrough-thickness", String => "string",
+    Stroke => "stroke", StrokeDasharray => "stroke-dasharray", StrokeDashoffset => "stroke-dashoffset",
```

Comment:
`stroke-*` classes - StrokeDash**A**rray, StrokeDash**O**ffset...   I think it would be more readable and I think I will be making typos in them. Just idea, I haven't used them yet.

Reply:
> I agree with the readability point.

I think readability is the top priority for source code. So I vote for human-friendly version - we can always write better generators.

I'm looking again at those classes - we can also respect `-`s and replace them with `_` if it's more readable.

@David-OConnor What's your opinion?

Reply:
I see benefits in both perspectives, but agree with Martin - going straight CamelCase is consistent with Rust styling, which the rest of this lib adheres to. I think in cases like this, we should design the API the way we want it, without worrying about previous standards.

On the fence over straight CamelCase vs including both options...  API pureness vs flexibility. Eg we could include the `StrokeDasharray` ones, but not document or recommend using them. See later on if anyone relies on them.

Reply:
What are y'alls thoughts on having both available?

Reply:
I don't having strong feelings about this, but I prefer only having one option. It enforces better consistency and does not pollute IDEs autocomplete with different options of the same thing.

Reply:
I'm strongly against multiple versions. It would be very ugly in IDE's autocomplete and once somebody see them we can just wait for new issues like `what attribute we should use?`. And I don't want to think about it as the user.

Reply:
@tiberiusferreira  - If you update this PR with Martin's changes, we can merge this.

Reply:
Related: What do y'all think of having `St` enum: An equivalent to `At`, but for styles? The groundwork's been there in a WIP state for a while; ended up not continuing due to the [large number of possible styles](https://developer.mozilla.org/en-US/docs/Web/CSS/Reference). I don't think it would hurt anything, but haven't heard a suggestion about this? The change would be to fill out [the make_styles!](https://github.com/David-OConnor/seed/blob/master/src/dom_types.rs#L340) macro call, add `St` to the prelude, and document it.

Reply:
I really like the idea, but I wonder if the increase in binary size will be negligible, after all the text will be stored (uncompressed before gzip, I imagine) somewhere in the binary. Anyway, I'd rather have the convenience even if at the cost of some KBs.

Reply:
Let's continue in https://github.com/David-OConnor/seed/issues/211
I think we can merge this PR now.